aten/src/ATen/native/native_functions.yaml

```diff
@@ -3557,26 +3557,26 @@
   use_c10_dispatcher: full
   variants: function, method
   dispatch:
-    QuantizedCPU: q_per_channel_scales_quant
+    QuantizedCPU: q_per_channel_scales_quantized_cpu
```

Comment:
these should be the same for all quantized tensors, so it should probably be named as `q_per_channel_scales_quantized`, similar for other operators that queries quantization parameters

Reply:
I think we can just use the same name for these ops: you can just use `q_per_channel_scales` for this one

Reply:
q_per_channel_scales